docs/Compose.md

```diff
@@ -0,0 +1,181 @@
+# Running GUAC with Docker Compose
+
+This tutorial introduces the full GUAC ingestion pipeline and how to deploy it
+with the included Docker Compose configuration.
```

Comment:
perhaps have a separate repo set up with git clone in a separate md linking to it as pre-requisite